apps/tx-signer/env/.env.sample

```
@@ -0,0 +1,13 @@
+PORT=
+SERVER_ORIGIN=
+RPC_NODE_ENDPOINT=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RPC_NODE_ENDPOINT key should go before the SERVER_ORIGIN key

(UnorderedKey)